andela/travel_tool_front
# #160195740 Display Request Details On Modal

#### What does this PR do?
This PR ensures that a users request detail is correctly displayed(dynamically) in the request detail modal

#### Description of Task to be completed?
- consume API to fetch data from the backend
- create reducer to listen to the dispatched action
- display data in request modal

#### How should this be manually tested?
- Clone the repo - https://github.com/andela/travel_tool_front.git
- Switch to the repo directory - `cd travel_tool_front`
- Install dependencies and start the app - `make start`
- Log in, click Requests tab on the left sidebar and select My Request
- Click on the Request Id on the table, then you should see modal displaying details of your request

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#160195740](https://www.pivotaltracker.com/story/show/160195740)

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2018-09-05 at 8 25 11 am" src="https://user-images.githubusercontent.com/26988297/45077502-41a2fc80-b0e5-11e8-8679-94613c5f429c.png">



#### Questions:
